src/components/com_tc/administrator/controllers/content.php

```diff
@@ -61,4 +64,65 @@ public function checkDuplicateAndLatestVersionTC()
 
 		jexit();
 	}
+
+	/**
+		* Function to save field data
+		*
+		* @param   string  $key     key
+		* @param   string  $urlVar  urlVar
+		*
+		* @return  boolean|void
+		*/
+	public function save($key = null, $urlVar = null)
+	{
```

Comment:
ensure this is similar to https://github.com/manojLondhe/joomla-extensions-generator/blob/master/templates/component/administrator/controllers/form.php